src/applications/hca/actions.js

```diff
@@ -105,7 +108,10 @@ function callAPI(dispatch, formData = {}) {
 // /health_care_applications/enrollment_status endpoint, depending on the value
 // of the `simulateServerLocally` flag
 export function getEnrollmentStatus(formData) {
-  return dispatch => {
+  return (dispatch, getState) => {
```

Comment:
It seems like we might want to make this a standard practice going forward?

Reply:
I personally like this, but I previously heard we discourage using the `getState` parameter. It doesn't look like it's documented in our [Redux guidelines](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/front-end-standards/redux) though. Curious about VSP's thoughts.

Reply:
Yeah- i think this is exactly the use case for `getState` being available to thunks.

Reply:
Dan Abramov is cool with it :-P https://stackoverflow.com/questions/35667249/accessing-redux-state-in-an-action-creator/35674575#35674575